### PR TITLE
updating balance calculation and correct colour for coloumns

### PR DESCRIPTION
**Pull request by elena-vi**

### CODE REVIEW

**Comment by elena-vi** [comment]
[reorganise account rent figures in the transaction history](https://trello.com/c/h6ir1PdH/89-reorganise-account-rent-figures-in-the-transaction-history)

File: spec/features/view_transactions_spec.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  end
+
+  def then_i_should_see_transaction_history
+    expect(page.body).to have_css('h2', text: 'Payment history', count: 1)
+    expect(page.body).to have_css('th', text: 'Date', count: 1)
+    expect(page.body).to have_css('th', text: 'Transaction', count: 1)
+    expect(page.body).to have_css('th', text: 'Code', count: 1)
+    expect(page.body).to have_css('th', text: 'Incoming', count: 1)
+    expect(page.body).to have_css('th', text: 'Outgoing', count: 1)
+    expect(page.body).to have_css('th', text: 'Balance', count: 1)
```
Comment (cob16) [inline spec/features/view_transactions_spec.rb]: 👍

File: lib/hackney/income/transactions_balance_calculator.rb [hunk 2]
```diff
@@ -10,12 +10,13 @@ def organise_with_final_balances_by_week(current_balance:, transactions:)
           incoming_sum = get_incoming_sum(transactions_in_week)
           outgoing_sum = get_outgoing_sum(transactions_in_week)
 
+          next unless transactions_in_week.any?
```
[the remark this answers is not in the document]
Comment (elena-vi) [reply lib/hackney/income/transactions_balance_calculator.rb]: rubo cop suggested this

[the remark this answers is not in the document]
Comment (mrosel) [reply lib/hackney/income/transactions_balance_calculator.rb]: There is a better reason than just rubocop. It's based in complexity - prefered would be a class method but if we are using flow control like this and its a single case then `unless` is easier to understand as it avoids double negatives. Also a lot of ppl missread the `!` in `if !true`
More complex situations or straightforward booleans, definitely use ` if` (and never case)

[the remark this answers is not in the document]
Comment (cob16) [reply lib/hackney/income/transactions_balance_calculator.rb]: let's make it so rubocop doesn't suggest this

**Pull request merged**